Troubleshoot/TroubleshootError.ps1

```diff
@@ -166,198 +184,236 @@ if ($account.Account -eq $null) {
 Write-Host("Checking resource group details...")
 Get-AzureRmResourceGroup -Name $ResourceGroupName -ErrorVariable notPresent -ErrorAction SilentlyContinue
 if ($notPresent) {
-	Write-Host("")
-	Write-Host("Could not find RG. Please make sure that the resource group name: '" + $ResourceGroupName + "'is correct and you have access to the cluster") -ForegroundColor Red
-	Write-Host("")
-	Stop-Transcript
-	exit
+    Write-Host("")
+    Write-Host("Could not find RG. Please make sure that the resource group name: '" + $ResourceGroupName + "'is correct and you have access to the Resource Group") -ForegroundColor Red
+    Write-Host("")
+    Stop-Transcript
+    exit
 }
 Write-Host("Successfully checked resource groups details...") -ForegroundColor Green
 
 Write-Host("Checking AKS Cluster details...")
 try {
-	$ResourceDetailsArray = Get-AzureRmResource -ResourceGroupName $ResourceGroupName -Name $AKSClusterName -ExpandProperties -ErrorAction Stop -WarningAction Stop
-} catch {
-	Write-Host("")
-	Write-Host("Could not fetch cluster details: Please make sure that the AKS Cluster name: '" + $AKSClusterName + "' is correct and you have access to the cluster") -ForegroundColor Red
-	Write-Host("")
-	Stop-Transcript
-	exit
+    $ResourceDetailsArray = Get-AzureRmResource -ResourceGroupName $ResourceGroupName -Name $AKSClusterName -ResourceType "Microsoft.ContainerService/managedClusters" -ExpandProperties -ErrorAction Stop -WarningAction Stop
+}
+catch {
+    Write-Host("")
+    Write-Host("Could not fetch cluster details: Please make sure that the AKS Cluster name: '" + $AKSClusterName + "' is correct and you have access to the cluster") -ForegroundColor Red
+    Write-Host("")
+    Stop-Transcript
+    exit
 }
 
 if ($ResourceDetailsArray -eq $null) {
-	Write-Host("")
-	Write-Host("Could not fetch cluster details: Please make sure that the AKS Cluster name: '" + $AKSClusterName + "' is correct and you have access to the cluster") -ForegroundColor Red
-	Write-Host("")
-	Stop-Transcript
-	exit
-} else {
-	Write-Host("Successfully checked AKS Cluster details...") -ForegroundColor Green
-	Write-Host("")
-	foreach ($ResourceDetail in $ResourceDetailsArray) {
-		if ($ResourceDetail.ResourceType -eq "Microsoft.ContainerService/managedClusters") {
-			$LogAnalyticsWorkspaceResourceID = $ResourceDetail.Properties.addonProfiles.omsagent.config.logAnalyticsWorkspaceResourceID
-			break
-		}
-	}
+    Write-Host("")
+    Write-Host("Could not fetch cluster details: Please make sure that the AKS Cluster name: '" + $AKSClusterName + "' is correct and you have access to the cluster") -ForegroundColor Red
+    Write-Host("")
+    Stop-Transcript
+    exit
+}
+else {
+    Write-Host("Successfully checked AKS Cluster details...") -ForegroundColor Green
```

Comment:
Printing the full cluster id will be helpful. Similarly for the log analytics workspace id.

Reply:
> Printing the full cluster id will be helpful. Similarly for the log analytics workspace id.

Its there as well.